Gemfile

```diff
@@ -164,6 +164,7 @@ end
 
 group :replication, :manageiq_default do
   gem "pg-pglogical",                   "~>2.1.2",       :require => false
+  gem "pg-logical_replication", "~>0.1", :git => "https://github.com/ManageIQ/pg-logical_replication.git", :require => false, :branch => "master"
```

Comment:
I assume by the time of merge, this will be released?

Reply:
That's the plan